### PR TITLE
[3.10] Deprecate some JHtmlBehavior methods

### PR DESCRIPTION
**Pull request by SharkyKZ**
# Summary of Changes

Marks methods to be removed in https://github.com/joomla/joomla-cms/pull/27335 as deprecated.

### Testing Instructions

Code review.

### Documentation Changes Required

Some methods deprecated.

### CODE REVIEW

**Comment by zero-24** [comment]
LGTM but lets wait for the 4.0 PR to be merged first. Please ping me when this happend and I didn't noticed. :-)

**Review by zero-24 — APPROVED** [review]
LGTM

File: libraries/cms/html/behavior.php [hunk 1]
```diff
@@ -280,6 +282,8 @@ public static function combobox()
 	 * @return  void
 	 *
 	 * @since   1.5
+	 *
+	 * @deprecated   4.0
```
Comment (wilsonge) [inline libraries/cms/html/behavior.php]: in favour of jhtml bootstrap.tooltip

File: libraries/cms/html/behavior.php [hunk 2]
```diff
@@ -204,6 +204,8 @@ public static function formvalidator()
...
```
Comment (wilsonge) [inline libraries/cms/html/behavior.php]: No replacement - only used in Hathor

File: libraries/cms/html/behavior.php [hunk 3]
```diff
@@ -511,6 +515,8 @@ public static function multiselect($id = 'adminForm')
...
```
Comment (wilsonge) [inline libraries/cms/html/behavior.php]: No replacement - seems unused in j3

**Comment by wilsonge** [comment]
@zero-24 merged in j4

File: libraries/cms/html/behavior.php (hunk 1 above)
Comment (SharkyKZ) [reply libraries/cms/html/behavior.php]: Added.

**Comment by zero-24** [comment]
Merging thanks :)

**Pull request merged**